Content.Client/Chat/UI/EmotesMenu.xaml.cs

```diff
@@ -46,13 +48,29 @@ public EmotesMenu()
                 !speech.AllowedEmotes.Contains(emote.ID))
                 continue;
 
+            string name;
+
+            if (emote.Category == EmoteCategory.GlobalCustom)
```

Comment:
Make a separate property on the emote prototype for "which CVar to read to replace contents" IMO, instead of checking the category like this. It would also fix the implicit and confusing behavior of the prototype ID having to match the CVar.